app/components/embed/footer_component.rb

```diff
@@ -4,8 +4,10 @@ module Embed
   class FooterComponent < ViewComponent::Base
     def initialize(viewer:)
       @viewer = viewer
+      @file_count = @viewer.purl_object.downloadable_files.length
+      @download_file_label = I18n.t :download_files, count: @file_count
```

Comment:
Why is this an ivar rather than a method call?

Reply:
Both of these values hang off the viewer, so both seem ripe for ditching the ivar and replacing the `attr_reader`s with methods.